CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Add tests for email feedback form
```

Comment:
Can you add the PR number as per the other entries?

Reply:
Sure thing, done :)